### ISSUE TITLE
Editing link with LinkBubbleMenu leads to outer form submission

### ISSUE THREAD

**Issue by PieceOfMeat** [sjdemartini/mui-tiptap#105]
### Describe the bug

If RichTextEditor with LinkBubbleMenu placed into an HTML form, editing link and hitting "Save" button leads to submission of this form.
I believe this happens because even despite in HTML these two forms are not connected, React bubbles submit event up through Portal to outer form component.

### To Reproduce

Use `<RichTextEditor>` inside a form, start editing a link, then hit "Save" button.

```javascript
const TestLinkEdit = () => {
  const handleSubmit = () => {
    alert("Gotcha!");
  };

  return (
    <form onSubmit={handleSubmit}>
      <RichTextEditor
        extensions={[StarterKit, Link, LinkBubbleMenuHandler]}
        content="<p>Hello world</p>"
        renderControls={() => (
          <MenuControlsContainer>
            <MenuButtonEditLink />
          </MenuControlsContainer>
        )}
      >
        {() => <LinkBubbleMenu />}
      </RichTextEditor>
    </form>
  );
};
```

### Expected behavior

Saving a link from pop-up form doesn't submit outer form.

### System:

- mui-tiptap version: 1.0.0
- tiptap version: 2.0.4

### Additional context

Currently workaround is to wrap `<LinkBubbleMenu />` with a dummy form, catching submit event. However, it leads to incorrect HTML structure ("form cannot appear as a descendant of form")
```javascript
<RichTextEditor {...}>
  {() => (
    <form onSubmit={(e) => e.stopPropagation()}>
      <LinkBubbleMenu />
    </form>
  )}
</RichTextEditor>
```

**Comment by sjdemartini** [comment]
Great catch @PieceOfMeat! Thank you for the bug report and helpful repro case. I've got a fix opened and plan to release it soon.

**Comment by sjdemartini** [comment]
The fix for this is part of the [v1.1.0 release](https://github.com/sjdemartini/mui-tiptap/tree/v1.1.0) I just published. Thanks again for the report!

**Comment by PieceOfMeat** [comment]
Thank you for a quick fix and a great package @sjdemartini :)
